### PR TITLE
Add TimeSinceLastFalse, TimeSinceLastMax, TimeSinceLastMin, TimeSinceLastTrue aggregation primitives

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: featuretools/primitives/standard/aggregation/time_since_last_true.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        ...          datetime(2010, 1, 1, 11, 57, 30)]
+        >>> booleans = [True, True, False]
+        >>> time_since_last_true(times, booleans, time=cutoff_time)
+        285.0
+    """
+
+    name = "time_since_last_true"
+    input_types = [
+        ColumnSchema(logical_type=Datetime, semantic_tags={"time_index"}),
+        ColumnSchema(logical_type=Boolean),
```
Comment (thehomebrewnerd) [inline featuretools/primitives/standard/aggregation/time_since_last_true.py:32]: Not sure if this should be done in a separate PR, but maybe want to update this and TimeSinceLastFalse to also work with BooleanNullable inputs?

File: featuretools/primitives/standard/aggregation/time_since_last_true.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    def get_function(self):
+        def time_since_last_true(datetime_col, bool_col, time=None):
+            df = pd.DataFrame(
+                {
+                    "datetime": datetime_col,
+                    "bool": bool_col,
+                },
+            ).dropna()
+            if df.empty:
+                return np.nan
```
Comment (thehomebrewnerd) [inline featuretools/primitives/standard/aggregation/time_since_last_true.py:48]: Not sure if it is critical, but should we add a test case to cover this line and the corresponding uncovered line in TimeSinceLastFalse?

Comment (gsheni) [reply featuretools/primitives/standard/aggregation/time_since_last_true.py:48]: Added e6d22c1b46206ed53a3f

File: featuretools/primitives/standard/aggregation/time_since_last_false.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        def time_since_last_false(datetime_col, bool_col, time=None):
+            df = pd.DataFrame(
+                {
+                    "datetime": datetime_col,
+                    "bool": bool_col,
+                },
+            ).dropna()
+            if df.empty:
+                return np.nan
+            df["bool"] = df["bool"].astype("bool")
```
Comment (thehomebrewnerd) [inline featuretools/primitives/standard/aggregation/time_since_last_false.py:55]: I think this line will cause problems with `BooleanNullable` inputs. However, I"m not real sure this is even necessary because the boolean column should always come in as either `bool` or `boolean` since we expect WW to be initialized on the series.

Comment (gsheni) [reply featuretools/primitives/standard/aggregation/time_since_last_false.py]: Removed https://github.com/alteryx/featuretools/pull/2418/commits/71390a33ab03f681a5cd4e8e34397e2a12ae909e

**Review by thehomebrewnerd — APPROVED** [review]
